resources/files/ruby_243/rbconfig/rbconfig-i386-pc-solaris2.10.rb

```diff
@@ -3,16 +3,16 @@
 # changes made to this file will be lost the next time ruby is built.
 
 module RbConfig
-  RUBY_VERSION == "2.4.2" or
-    raise "ruby lib version (2.4.2) doesn't match executable version (#{RUBY_VERSION})"
+  RUBY_VERSION == "2.4.3" or
+    raise "ruby lib version (2.4.3) doesn't match executable version (#{RUBY_VERSION})"
 
   TOPDIR = File.dirname(__FILE__).chomp!("/lib/ruby/2.4.0/i386-solaris2.10")
   DESTDIR = '' unless defined? DESTDIR
   CONFIG = {}
   CONFIG["DESTDIR"] = DESTDIR
   CONFIG["MAJOR"] = "2"
   CONFIG["MINOR"] = "4"
-  CONFIG["TEENY"] = "2"
+  CONFIG["TEENY"] = "3"
   CONFIG["PATCHLEVEL"] = "111"
```

Comment:
This is a nitpick, but we might want to consider bumping PATCHLEVEL and RUBY_RELEASE per https://github.com/ruby/ruby/blob/v2_4_3/version.h#L2-L3.